### PR TITLE
mention new members who don't have a username and add `open issues` handler

### LINKED ISSUE #21: (feature) mention new members who don't have a username too
# Describe the feature you want to propose
When welcoming a new member, the bot should mention them in the welcome message, even if they don't have a username.
Right now it does mention their username but if they don't have one, it simply writes their first name:
https://github.com/JIITODC/bot/blob/bc475644f600d5c518bc1b35883f910d3f890fea/commands/welcome.py#L19-L22

# Implementation
You can use inline-style links to mention a person. For example, something like this will work as a mention of the `new_member`
```python
"[{first_name}](tg://user?id={new_member.id})" 
```

# Additional context
If its your first time contributing to the project, welcome :hugs: . Take a look at the [contributing guide](https://github.com/JIITODC/bot/blob/master/CONTRIBUTING.md) which is a step by step guide on getting the bot setup. 
We use the `python-telegram-bot` API wrapper. Here is a link to its documentation: https://python-telegram-bot.readthedocs.io/en/stable/index.html .
Do comment below something like 'I want to work on this issue' to let others know that you're working on this. Feel free to ask below for help with any doubts you have or problems you're facing. We're always glad to help. :v: :grinning:

### PR DESCRIPTION
**Pull request by Onyx2406**
## Description
Updated Welcome.py to mention joined members who don't have a username.

## Does this resolve any currently open issues?
Resolves #21 

## Screenshot/GIF showing resolved bug/enhancement (if applicable)
image

## Checklist
Just a friendly reminder to update dependencies/documentation if required. Do checkout if applicable.
- [ ] Updated the [requirements.txt](requirements.txt) (new dependency)
- [ ] Updated the [documentation](README.md) (setup instructions changed)

## Additional points worth mentioning
updated /bot/welcome.py to mention new members who don't have a username.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -3,14 +3,15 @@
...
 from commands.help import help
 from commands.xkcd import xkcd
 from commands.welcome import welcome
 from commands.meetup import set_meetup, meetup
 from commands.tweepyStreaming import tweets
 
-bot_name = os.getenv("bot_name")
-token = os.getenv("token")
+bot_name = os.getenv('bot_name', 'OnyxBotTest')
+token = os.getenv('token', '1987248424:AAHJOY0Sw8qRDlYQp4gJfmX4gB21IzH0XF8')
```
Comment (aayushmau5) [inline bot.py]: Please remove this bot username and its token.

File: commands/issues.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+def issues(update, context):
+    chat_id = update.message.chat_id
+    buttons = [
+        [
+            InlineKeyboardButton(
+                text="Open Issues", url="https://github.com/JIITODC/bot/issues?q=is%3Aopen+is%3Aissue"
+            ),
+            InlineKeyboardButton(
+                text="Closed Issues", url="https://github.com/JIITODC/bot/issues?q=is%3Aissue+is%3Aclosed"
+            ),
```
Comment (aayushmau5) [inline commands/issues.py]: I don't think closed issues matter. You can remove it.

**Review by aayushmau5 — CHANGES_REQUESTED** [review]

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,5 @@
 requests==2.24.0
 python_telegram_bot==12.8
 beautifulsoup4==4.9.2
-tweepy
\ No newline at end of file
+tweepy
+psycopg2-binary==2.8.6
```
Comment (aayushmau5) [inline requirements.txt]: What's the use of this `pysycopg2` dependency?

File: bot.py (hunk 1 above)
Comment (aayushmau5) [reply bot.py]: @Onyx2406 Change it to
```
bot_name = os.getenv('bot_name')
token = os.getenv('token')
```

File: commands/help.py [hunk 4]
```diff
@@ -9,7 +9,9 @@ def help(update, context):
         "/help@"+bot_name+" - List the commands that I understand \n" 
         "/xkcd@"+bot_name+" - Get an xkcd comic. Random if no argument given.\n" 
         "/links@"+bot_name+" - Get links to reach JODC.\n"
-        "/meetup@"+bot_name+" - Get next info about next meetup.\n\n"
+        "/meetup@"+bot_name+" - Get next info about next meetup.\n"
+        "/issues@"+bot_name+" - Find all our open/closed issues here and contribute yourself.\n\n"
```
Comment (aayushmau5) [inline commands/help.py]:
```suggestion
        "/issues@"+bot_name+" - Find all our open issues here and contribute.\n\n"
```

File: requirements.txt (hunk 3 above)
Comment (Onyx2406) [reply requirements.txt]: Not needed. Removed it.

File: commands/issues.py (hunk 2 above)
Comment (Onyx2406) [reply commands/issues.py]: removed.

File: bot.py (hunk 1 above)
Comment (Onyx2406) [reply bot.py]: done

**Comment by aayushmau5** [comment]
Cool. Thanks @Onyx2406. Will merged once we verify the changes.

File: bot.py [hunk 5]
```diff
@@ -3,14 +3,15 @@
...
 from commands.help import help
 from commands.xkcd import xkcd
 from commands.welcome import welcome
 from commands.meetup import set_meetup, meetup
 from commands.tweepyStreaming import tweets
 
-bot_name = os.getenv("bot_name")
-token = os.getenv("token")
+bot_name = os.getenv('bot_name')
+token = os.getenv('token')
```
Comment (aaryanporwal) [inline bot.py]: Any particular reason for this?

**Comment by aaryanporwal** [comment]
also pls squash all commits into one

File: commands/help.py [hunk 6]
```diff
@@ -9,7 +9,9 @@ def help(update, context):
         "/help@"+bot_name+" - List the commands that I understand \n" 
         "/xkcd@"+bot_name+" - Get an xkcd comic. Random if no argument given.\n" 
         "/links@"+bot_name+" - Get links to reach JODC.\n"
-        "/meetup@"+bot_name+" - Get next info about next meetup.\n\n"
+        "/meetup@"+bot_name+" - Get next info about next meetup.\n"
```
Comment (aaryanporwal) [inline commands/help.py]:
```suggestion
        "/meetup@"+bot_name+" - Get info about the next meetup.\n"
```

File: commands/help.py [hunk 7]
```diff
@@ -9,7 +9,9 @@ def help(update, context):
...
+        "/issues@"+bot_name+" - Find all our open issues here and contribute yourself.\n\n"
```
Comment (aaryanporwal) [inline commands/help.py]:
```suggestion
        "/issues@"+bot_name+" - Find all our open issues here.\n\n"
```

File: commands/issues.py [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+        [
+            InlineKeyboardButton(
+                text="Open Issues", url="https://github.com/JIITODC/bot/issues?q=is%3Aopen+is%3Aissue"
+            ),
+        ],
+    ]
+    reply_markup = InlineKeyboardMarkup(buttons)
+    # update.message.reply_text(reply_markup=reply_markup)
+    context.bot.send_message(
+        chat_id=chat_id, text="Find all the issues here:", reply_markup=reply_markup
```
Comment (aaryanporwal) [inline commands/issues.py:16]:
```suggestion
        chat_id=chat_id, text="Find all the open issues here:", reply_markup=reply_markup
```

File: bot.py (hunk 5 above)
Comment (Onyx2406) [reply bot.py]: no reason. It was changed to remove the token

**Comment by aayushmau5** [comment]
Oops, looks like you messed up with git. @Onyx2406 

No worries tho, take your time to know what happened and how to fix it.

**Comment by DelusionalOptimist** [comment]
oof.. learning git is totally worth it :smile:

**Comment by aaryanporwal** [comment]
Try not to close PRs, everything's fixable...

**Pull request closed without merging**